linkedList.js

```diff
@@ -0,0 +1,372 @@
+class Node {
+  constructor(data) {
+    this._data = data;
+    this.nextNode = null;
+  }
+
+  getData() {
+    return this._data;
+  }
+
+
+}
+
+class LinkedList {
+  constructor() {
+    this.head = null;
+  }
+
+  length() {
+    var currentNode = this.head;
+    var length = 0;
+
+    if(!currentNode) {
+      return null;
```

Comment:
In this case, this.head is null and the list is empty. So the length returned should be 0 and not null.